docs/altblocktree.md

```diff
@@ -0,0 +1,132 @@
+# AltBlockTree {#altblocktree}
+
+Represents simplified view on Altchain's block tree, maintains VBK tree and
+BTC tree. VBK blocks and BTC blocks are added through Payloads (VTB, VbkBlock).
+
+Helps Altchain to compare Altchain blocks by POP score (adding Endorsements
+in VBK chain increases POP score) in POP Fork Resolution and to calculate POP
+Rewards - payout to POP miners.
+
+AltBlockTree is initialized with Altchain, Veriblock and Bitcoin parameters, as well as with PayloadsProvider implementation.
+```c++
+// use mainnet for all chains, for instance
+AltChainParamsMain altp;
+VbkChainParamsMain vbkp;
+BtcChainParamsMain btcp;
+// your implementation of PayloadsProvider
+PayloadsProviderImpl provider;
+AltBlockTree tree(altp, vbkp, btcp, provider);
+```
+
+After initialization AltBlockTree does not contain any blocks. 
+**Users MUST bootstrap AltBlockTree** - add initial/genesis/root block of Altchain.
+
+We will refer to this block as **bootstrap block**.
```

Comment:
I think we support multiple bootstrap blocks.

Reply:
In Alt tree - only one.